MBBSEmu/Configuration.cs

```diff
@@ -139,17 +142,27 @@ public T GetAppSettings<T>(object value, string valueName)
             }
         }
 
-        private string GetStringAppSettings(string key)
+        private string GetFileNameAppSettings(string key)
         {
-            if (string.IsNullOrEmpty(ConfigurationRoot[key]))
+            //strip paths
+            var pathFile = Path.GetFileName(ConfigurationRoot[key]);
+
+            if (!string.IsNullOrEmpty(pathFile))
+                return pathFile;
+
+            switch (key)
             {
-                throw key switch
-                {
-                    "Database.File" => new Exception($"Please set a valid database filename(eg: mbbsemu.db) in the {Program._settingsFileName ?? Program.DefaultEmuSettingsFilename} file before running MBBSEmu"),
-                    _ => new Exception($"Missing {key} in {Program._settingsFileName ?? Program.DefaultEmuSettingsFilename}"),
-                };
+                case "Database.File":
+                    _logger.Warn($"No valid database filename set in {Program._settingsFileName ?? Program.DefaultEmuSettingsFilename} -- setting default value: mbbsemu.db");
+                    pathFile = "mbbsemu.db";
+                    return pathFile;
+                case "File.LogName":
+                    _logger.Info($"No valid filename for logging, disabling");
```

Comment:
I'm torn if we'll need this message. For example, I'll have it off forever because file logging is synchronous and murders my local performance when debugging.

Reply:
Removed message, agree -- if someone wants file logging they will put it in there we don't need to warn them if they don't want it in the first place.